ci_test.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Prep the results.markdown file
+echo "Results for "$JOB_NAME > results.markdown
+echo "" >> results.markdown
+echo 'Module | Pass/Fail' >> results.markdown
+echo '-----|-------|' >> results.markdown
+
+# Create python3 virt env for the test
+python3 -m venv ci_test
+
+source ci_test/bin/activate
+
+# Install scribe into the venv
+pip3 install -e .
+
+rc=0
+
+# Test each module in the ci folder
+for module in `ls ci`
```

Comment:
@aakarshg we loop through the json files here. The idea is that if we don't actually pass scribe real data it will just run and never actually try any of the modules so the ci pass would be meaningless. So here we have a 1-to-1 of module to json test file. We can then accurately say yes/no this module worked on that json.

Reply:
So everytime stockpile is updated we'd have to update these jsons? Don't think that'll work out long term, instead what i'll suggest is we run stockpile job first and run scribe against those module, that's how we previously tested too. This would be helpful as it'll show us the end results.

Reply:
By running stockpile, we'll also be actually dealing with real data, instead of using the old generated data which might/might not be accurate.

Reply:
Thats fine except if we blankly run stockpile on the ci system it may not use all the modules that we need to test against. I.e. if its an openstack module or something. This was my attempt to work around that but I'm open to other ideas.

Reply:
agreed currently ci system may not use the openstack specific modules, but think its better than having to maintain these jsons, as they may not always provide us with the right idea i.e. less confidence imho. About the ci system missing openstack, it can easily be worked by doing a shift on stack environment, but not a priority right now as openstack modules run in browbeat-ci and if there's any issue we'd get know sooner than later :)